### DISCUSSION TITLE
Thank you 🤗 and link to a similar tool

### DISCUSSION THREAD

**Discussion by piotr-yuxuan** [sunng87/diehard#50] [General]
Hello, thank you for this great little project! I wonder if you've come across safely [0] and whether you could find ideas to port to diehard :-)

[0] https://github.com/BrunoBonacci/safely

**Comment by sunng87** [comment]
@piotr-yuxuan Thank you 雨轩！Since this is not an issue I'm moving this into discussions. safely looks pretty similar to diehard. Its integration with mulog seems promising (from the same author). Always let me know if you find something missing from diehard that can be added.
